src/common/Extensions/HttpExtensions.cs

```diff
@@ -7,7 +7,6 @@
 using Microsoft.BridgeToKubernetes.Common;
 using Microsoft.BridgeToKubernetes.Common.Json;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace System.Net.Http

```

Comment:
@Eneuman I don't see this class file is used anywhere can we remove this, or I saw another PR coming for unused code is it covered in that ?

Reply:
@Eneuman If they are use then it is fine.